core/src/main/java/org/apache/iceberg/ManifestGroup.java

```diff
@@ -154,6 +156,12 @@ ManifestGroup caseSensitive(boolean newCaseSensitive) {
     return this;
   }
 
+  ManifestGroup columnsToKeepStats(Set<Integer> newColumnsToKeepStats) {
+    this.columnsToKeepStats =
+        newColumnsToKeepStats == null ? null : Sets.newHashSet(newColumnsToKeepStats);
```

Comment:
This copy seems redundant but up to you.

Reply:
Kept as it is more consistent with the other implementations